cloud

```diff
@@ -2267,7 +2272,7 @@ function execute() {
         fi
         if [[ $command == @(deploy|env|shell) ]]; then
             getopt_longopts+=",extra-cc:,smtp-to-addrs:"
-            getopt_longopts+=",log-level:,optimize:,heavy-asserts,mute-updates"
+            getopt_longopts+=",log-level:,optimize:,cache-bucket-name:,heavy-asserts,mute-updates"
```

Comment:
We don't need this here, since `cloud` is not taking the option.

Reply:
okay